packages/server/src/data/db/db-operations/user-ops.js

```diff
@@ -9,6 +9,24 @@ const createUser = (payload) => {
   return User.create(payload);
 };
 
+const updateUser = ({ _id, username, email }) => {
+  const updateResult = User.findByIdAndUpdate(
```

Comment:
Why not just return `User.findByIdAndUpdate

Reply:
change from https://github.com/jpfranci/Alpacow/pull/26